src/components/menu-bar/author-info.css

```diff
@@ -0,0 +1,51 @@
+@import "../../css/colors.css";
+@import "../../css/units.css";
+
+
+
+.author-info {
+    color: $ui-white;
+    font-family: "Helvetica Neue";
+    display: flex;
+    justify-content: start;
+    align-items: center;
+}
+
+.author-info .avatar {
+    margin-right: .5625rem;
+}
+
+.author-info .title-author {
+    /* display: flex;
+    align-items: center;
+    justify-content: end; */
+}
+
+.author-info .project-title {
+    max-width: 12rem;
+    display: block;
+    overflow: hidden;
+    font-size: .875rem;
+    line-height: 1.0625rem;
+    font-weight: bold;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+}
+
+.author-info .username-line {
+    max-width: 12rem;
+    font-size: .75rem;
+    line-height: .9375rem;
```

Comment:
Would any of these numbers make sense to move into `units` so they can be reused?

Reply:
Hm, not sure. I'll take a look at other menu bar css and see if there are repeated values

Reply:
Done, consolidated a bunch of them.